src/components/FilterDropdown.vue

```diff
@@ -151,14 +202,14 @@ import Popover from './Popover.vue';
 import { QueryFilter } from 'utils/db/types';
 
 const conditions = [
-  { label: t`Is`, value: '=' },
-  { label: t`Is Not`, value: '!=' },
-  { label: t`Contains`, value: 'like' },
-  { label: t`Does Not Contain`, value: 'not like' },
-  { label: t`Greater Than`, value: '>' },
-  { label: t`Less Than`, value: '<' },
-  { label: t`Is Empty`, value: 'is null' },
-  { label: t`Is Not Empty`, value: 'is not null' },
+  { label: '=', value: '=' },
+  { label: '!=', value: '!=' },
+  { label: 'like', value: 'like' },
+  { label: 'not like', value: 'not like' },
+  { label: '>', value: '>' },
+  { label: '<', value: '<' },
+  { label: 'is null', value: 'is null' },
+  { label: 'is not null', value: 'is not null' },
```

Comment:
Is this change really needed?. Labels are used to make the easy for the user for daily operations. For a non-technical user perspective, `!=` doesn't directly imply what it is used for.